maintainers/maintainer-list.nix

```diff
@@ -5164,6 +5164,12 @@
     githubId = 546087;
     name = "Kristoffer K. Føllesdal";
   };
+  kho-dialga = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add kho-dialga`` and sort it alphabetically.